modules/nw-pod-network-connectivity-implementation.adoc

```diff
@@ -47,6 +48,6 @@ spec:
 ----
 <1> Specifies the network diagnostics configuration. If a value is not specified or an empty object is specified, and `spec.disableNetworkDiagnostics=true` is set in the `network.operator.openshift.io` custom resource named `cluster`, network diagnostics are disabled. If set, this value overrides `spec.disableNetworkDiagnostics=true`.
 <2> Specifies the diagnostics mode. The value can be the empty string, `All`, or `Disabled`. The empty string is equivalent to specifying `All`.
-<3> Optional: Specifies a selector for connectivity check source pods. You can use the `nodeSelector` and `tolerations` fields to further specify the `sourceNode` pods. You do not have to use both `nodeSelector` and `tolerations`, however, for both the source and target pods. These are optional fields that can be omitted.
+<3> Optional: Specifies a selector for connectivity check source pods. You can use the `nodeSelector` and `tolerations` fields to further specify the `sourceNode` pods. While `nodeSelector`` and `tolerations` are optional for both source and target pods, you do not need to use both.
```

Comment:
I am not sure if I'd rephrase this like: "`nodeSelector` and `tolerations` are optional, and you don't need to use both" or "`nodeSelector` and `tolerations` are optional, so you don't need to use both".

The idea is that they are optional and you can use only one of them, or both of them, as you wish.